src/components/Common/ProposalPreviewCard/ProposalPreviewCard.tsx

```diff
@@ -63,45 +92,58 @@ const ProposalPreviewCard = ({ proposal, votes, variant }: Props) => {
+                )}
+                {showVotedChoice && (
+                  <span
+                    title={userChoice.toUpperCase()}
+                    className={classNames(
+                      'ProposalPreviewCard__DetailsItem',
+                      'ProposalPreviewCard__DetailsItemVotedChoice'
+                    )}
+                  >
+                    {t('page.proposal_detail.your_vote_label')}
+                    <span className="ProposalPreviewCard__Vote">{userChoice}</span>
+                  </span>
+                )}
+                <Desktop>
+                  <span className="ProposalPreviewCard__DetailsItem">
+                    {t('page.home.open_proposals.votes', { total: Object.keys(votes || {}).length })}
+                  </span>
+                  <span className="ProposalPreviewCard__DetailsItem">
+                    {t('page.home.open_proposals.comments', { total: comments?.totalComments || 0 })}
+                  </span>
+                </Desktop>
+              </>
             )}
-            {showVotedChoice && (
+
+            {variant !== Variant.Slim ? (
               <span
-                title={userChoice.toUpperCase()}
                 className={classNames(
                   'ProposalPreviewCard__DetailsItem',
-                  'ProposalPreviewCard__DetailsItemVotedChoice'
+                  isProposalAboutToEnd && 'ProposalPreviewCard__DetailsItem--highlight'
                 )}
               >
-                {t('page.proposal_detail.your_vote_label')}
-                <span className="ProposalPreviewCard__Vote">{userChoice}</span>
-              </span>
-            )}
-            <Desktop>
-              <span className="ProposalPreviewCard__DetailsItem">
-                {t('page.home.open_proposals.votes', { total: Object.keys(votes || {}).length })}
+                {dateText}
               </span>
-              <span className="ProposalPreviewCard__DetailsItem">
-                {t('page.home.open_proposals.comments', { total: comments?.totalComments || 0 })}
+            ) : (
+              <span className={classNames('ProposalPreviewCard__DetailsItem')}>
+                {getPriorityText(proposal as PriorityProposal, t)}
               </span>
-            </Desktop>
-            <span
-              className={classNames(
-                'ProposalPreviewCard__DetailsItem',
-                isProposalAboutToEnd && 'ProposalPreviewCard__DetailsItem--highlight'
-              )}
-            >
-              {dateText}
-            </span>
+            )}
           </span>
         </div>
       </ProposalPreviewCardSection>
-      {variant === Variant.Vote && <VoteModule proposal={proposal} votes={votes} />}
-      {variant === Variant.Category && <CategoryModule proposal={proposal} isHovered={isHovered} />}
+      {variant === Variant.Vote && <VoteModule proposal={proposal as ProposalAttributes} votes={votes} />}
+      {variant !== Variant.Vote && (
```

Comment:
As this component is getting bigger maybe I'd be specific here on how these components are shown.
`const showCategoryModule = variant === Variant.Category && variant === Variant.Slim`